tap_zendesk/__init__.py

```diff
@@ -1,13 +1,16 @@
 #!/usr/bin/env python3
 import json
-import sys, os
-from datetime import datetime
-from zenpy import Zenpy
+import os
+import sys
+from time import sleep
+
 import requests
+import singer
 from requests import Session
 from requests.adapters import HTTPAdapter
-import singer
 from singer import metadata, Schema, metrics as singer_metrics
+from zenpy import Zenpy
```

Comment:
Why do you need this import?

Reply:
It was only reordered (pylint). This `Zenpy` is the class that the `tap-zendesk` use to interact with the API.